source/gpgpu.rst

```diff
@@ -0,0 +1,30 @@
+.. toctree::
+
+GPGPUs on EGI Cloud
+===================
+
+GPGPUs resources are available on selected providers of the EGI Cloud. These 
+are available as specific **flavors** that when used to instantiate a Virtual
+Machine will make the hardware available for the user.
```

Comment:
```suggestion
Machine will make the hardware available to the user.
```

Reply:
https://forum.wordreference.com/threads/available-to-available-for.2132015/#post-10671203